model-optimizer/unit_tests/extensions/ops/split_test.py

```diff
@@ -188,3 +188,76 @@ def test_splitv_2_zero_not_last(self):
         self.assertTrue(node.out_port(4).disconnected())
         self.assertTrue(node.out_port(3).disconnected())
         self.assertTrue(np.all(node.split_lengths == np.array([2, 13, 10])))
+
+class TestVariadicSplitOp(unittest.TestCase):
```

Comment:
Tests are identical except the input shape of the axis. Please, generalize tests using with the generator.
Also, add a test which fails the shape inference, for example, shape of axis is `[2]` or `[1,1]`

Reply:
Done, unit tests updated.